src/chunk.c

```diff
@@ -4871,3 +4875,9 @@ ts_chunk_get_osm_chunk_id(int hypertable_id)
 
 	return chunk_id;
 }
+
+TimestampTz
+ts_chunk_get_creation_time(Oid chunk_oid)
+{
+	return GetCurrentTimestamp();
+}
```

Comment:
Leftover?

Reply:
Yes, this needs to integrated with follow up PR on policy implementation.